project/make.sh

```diff
@@ -58,6 +58,8 @@ DEFAULT_BUNDLES=(
 	cover
 	cross
 	tgz
+
+	fedora
```

Comment:
So, is this RPM Fedora-specific, or generic for all "RPM distributions" ?  I'd rather see us avoid the mistake we made with the current `.deb` where it's supposed to be generic and that limits us and causes bugs that we're going to have to split it out into multiple `.deb`s to fix (which is going to be painful both for us, and for our users).